scripts/open-in-repl.js

```diff
@@ -0,0 +1,33 @@
+/* global jQuery */
+
+(function ($) {
+    'use strict';
+
+    var presets = [ "es2015", "stage-0", "react" ];
```

Comment:
This is meant to be available on the entire website. I tried to be as compatible as possible while not including all presets...

I guess you could use the `package` value in the header to determine which preset is using in the examples.